geop/src/main/scala/com/azavea/usace/programanalysis/geop/GeopServiceActor.scala

```diff
@@ -49,16 +68,39 @@ class GeopServiceActor(sc: SparkContext) extends Actor with HttpService {
         complete {
           future {
             args.multiPolygons.map(m => {
-              val multiPolygon = m.reproject(LatLng, ConusAlbers)
-              val rasterLayers = ClippedLayers(args.rasters, multiPolygon, sc)
-              val rasterGroupedCount = RasterGroupedCount(rasterLayers, multiPolygon)
-
-              JsObject(
-                rasterGroupedCount
-                  .map { case (keys, count) =>
-                    keys.mkString(",") -> JsNumber(count)
-                  })
-              }).toVector
+                val multiPolygon = m.reproject(LatLng, ConusAlbers)
```

Comment:
Looks like this block was accidentally changed to use four spaces, and has no other changes.